browser/brave_browser_process_impl.cc

```diff
@@ -317,17 +315,6 @@ void BraveBrowserProcessImpl::OnTorEnabledChanged() {
 }
 #endif
 
```

Comment:
is there any kind of local state -> profile prefs migration that needs to happen?

Reply:
No need to migrate from local state to profile prefs, but we do want to clean-up after expired prefs, we have an open ticket just for that https://github.com/brave/brave-browser/issues/19502